source/adapter.js

```diff
@@ -346,7 +346,7 @@ AdapterJS.parseWebrtcDetectedBrowser = function () {
     webrtcDetectedBrowser   = 'safari';
     webrtcDetectedVersion   = parseInt(hasMatch[1] || '0', 10);
     webrtcMinimumVersion    = 7;
-    webrtcDetectedType      = isMobile.length === 0 ? 'plugin' : nullg;
+    webrtcDetectedType      = isMobile.length === 0 ? 'plugin' : null;
```

Comment:
Tested changes on Safari 10, 9 on MacOSX and Safari 8 on iOS. Values are reflected correctly.